pkgs/development/libraries/libredwg/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, texinfo, pcre2
+, enablePython ? false, python, swig, libxml2, ncurses
+}:
+let
+  isPython3 = enablePython && python.pythonAtLeast "3";
+in
+stdenv.mkDerivation rec {
+  pname = "libredwg";
+  version = "0.10.1";
+
+  src = fetchFromGitHub {
+    owner = "LibreDWG";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:1zd721z2nriw1jlrh4y1fj59b0dnymhd4kwp8rqw16bs84gda37n";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig texinfo ];
+
+  buildInputs = [ pcre2 ]
+    ++ lib.optionals enablePython [ python swig libxml2 libxml2.dev ]
```

Comment:
I would expect `swig` to be a build time dependency so it should go to `nativeBuildInputs`.